pipeline/Makefile

```diff
@@ -301,11 +314,11 @@ data/extracted/cta_train_ridership.csv:
 		--output_file="data/extracted/cta_train_ridership.csv"
 		
 
-data/extracted/daily_bus_rideship.csv:
+data/extracted/daily_bus_ridership.csv:
     python3 extract/from_data_portal.py \
-        --json_url="$(PORTAL_BUS_RIDESHIP)" \
-        --soql_file="extract/daily_bus_rideship.sql" \
-        --output_file="data/extracted/daily_bus_rideship.csv"
+        --json_url="$(PORTAL_BUS_RIDERSHIP)" \
```

Comment:
The spacing in this one looks different in comparison to the other steps under this section of the make file. Try to backspace all of the indented lines after line 317 to be in line with that line and then use tab to indent them